org.freedesktop.Sdk.Extension.ldc2.yml

```diff
@@ -0,0 +1,49 @@
+app-id: org.freedesktop.Sdk.Extension.ldc2
```

Comment:
Since there is no d1 compiler flatpak extension, and also upstream calls this one "ldc" (without 2), maybe the ID should be `.ldc`?

Reply:
The binaries are called ldc2 so I figured this would be clearer - but I guess ldc works too, I doubt ldc will release ldc3 any time soon.